dependencyLibs/Mavlink/src/com/MAVLink/Messages/ApmModes.java

```diff
@@ -30,6 +30,7 @@
 	ROTOR_SPORT(13, "Sport",MAV_TYPE.MAV_TYPE_QUADROTOR),
 	ROTOR_AUTOTUNE(15, "Autotune",MAV_TYPE.MAV_TYPE_QUADROTOR),
 	ROTOR_POSHOLD(16, "PosHold",MAV_TYPE.MAV_TYPE_QUADROTOR),
+	ROTOR_BRAKE(17,"Brake",MAV_TYPE.MAV_TYPE_QUADROTOR),
```

Comment:
@djmedina anything in the Mavlink project is automatically generated, and should not be edited manually.
To update the list of apm modes, you need to find and update the definition files defining the modes, and regenerate the mavlink library.